aerogear-android-push/src/main/java/org/jboss/aerogear/android/unifiedpush/gcm/AeroGearGCMPushRegistrar.java

```diff
@@ -51,16 +47,20 @@
     private final static String AUTHORIZATION_METHOD = "Basic";
 
     private static final Integer TIMEOUT = 30000;// 30 seconds
+    private static final String TAG = AeroGearGCMPushRegistrar.class.getSimpleName();
     /**
-     * Default lifespan (7 days) of a registration until it is considered
-     * expired.
+     * This template creates a key used by the registrar to save push data to SharedPreferences. 
+     * This information will be fetched by {@link UnifiedPushInstanceIDListenerService} 
+     * in the event registration IDs are reloaded.
      */
-    public static final long REGISTRATION_EXPIRY_TIME_MS = 1000 * 3600 * 24 * 7;
-
-    private static final String TAG = AeroGearGCMPushRegistrar.class.getSimpleName();
-    public static final String PROPERTY_REG_ID = "registration_id";
-    private static final String PROPERTY_APP_VERSION = "appVersion";
-    private static final String PROPERTY_ON_SERVER_EXPIRATION_TIME = "onServerExpirationTimeMs";
+    static final String REGISTRAR_PREFERENCE_PATTERN = "org.jboss.aerogear.android.unifiedpush.gcm.AeroGearGCMPushRegistrar:.+";
+    /**
+     * This template creates a key used by the registrar to save push data to SharedPreferences. 
+     * This information will be fetched by {@link UnifiedPushInstanceIDListenerService} 
+     * in the event registration IDs are reloaded.
+     */
```

Comment:
The pattern is a regular expression used by the shared preferences store to find token entries.
The template is used by the Registrar to create keys in shared preferences which match the token.  One is a pattern the other is a template.